src/content/en/2019/accessibility.md

```diff
@@ -245,40 +247,43 @@ The relative popularity of the [dialog role](https://developer.mozilla.org/en-US
+Similarly, assistive software is not only geared towards people with disabilities:
+- Drivers would benefit from a screen reading functionality that, while they keep their eyes on the road, reads long pieces of text like news stories aloud.
+- The recent popularity of voice assistance, both on mobile devices and in the home, has demonstrated that controlling a computing device using voice commands is desirable (and essential for many users). Voice commands like these used to only be an accessibility feature but are now turning into a mainstream product.
+- Captions are enjoyed not only by people who cannot hear a video but also by people who want to watch a video in a loud restaurant or in a library.
 
 Once a website is built, it’s often hard to retrofit accessibility on top of existing site structures and widgets. Accessibility isn’t something that can be easily sprinkled on afterwards, but needs to be part of the design and implementation process. Unfortunately, either through a lack of awareness or easy-to-use testing tools, many developers are not familiar with the needs of all their users and the requirements of the assistive software they use.
 
-While not conclusive, our results indicate that the use of accessibility standards like ARIA, and good accessibility authoring practices (e.g., the use of alt text) are found on a *sizable, but not substantial,* portion of the web. On the surface this is encouraging, but we suspect many of these positive trends are due to the popularity of certain UI frameworks. On one hand, this is disappointing because web developers cannot simply rely on UI frameworks to inject their sites with accessibility support. On the other hand though, it’s encouraging to see how large of an effect UI frameworks could have on the accessibility of the web.
+While not conclusive, our results indicate that the use of accessibility standards like ARIA, and accessibility best practices (e.g., using alt text) are found on a *sizable, but not substantial,* portion of the web. On the surface this is encouraging, but we suspect many of these positive trends are due to the popularity of certain UI frameworks. On one hand, this is disappointing because web developers cannot simply rely on UI frameworks to inject their sites with accessibility support. On the other hand though, it’s encouraging to see how large of an effect UI frameworks could have on the accessibility of the web.
```

Comment:
```suggestion
While not conclusive, our results indicate that the use of accessibility standards like ARIA and accessibility best practices (e.g., using alt text) are found on a *sizable, but not substantial* portion of the web. On the surface this is encouraging, but we suspect many of these positive trends are due to the popularity of certain UI frameworks. On one hand, this is disappointing because web developers cannot simply rely on UI frameworks to inject their sites with accessibility support. On the other hand though, it’s encouraging to see how large of an effect UI frameworks could have on the accessibility of the web.
```